src/applications/check-in/components/layout/tests/Footer.unit.spec.jsx

```diff
@@ -67,7 +52,7 @@ describe('check-in', () => {
       );
       expect(component.getByTestId('travel-help-block')).to.exist;
```

Comment:
this is testing a testId that is always present no matter if it is showing the travel info